src/jobsautoreport/process.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+
+from jobsautoreport.query import Querier
+
+
+class Processor:
```

Comment:
```suggestion
class Reporter:
```
I think it would make sense to name it this way as this class contains the main logic to construct a report from a list of jobs.